### ISSUE TITLE
Column example

### ISSUE THREAD

**Issue by tarlepp** [DataTorrent/malhar-angular-dashboard#114]
Hi, it would be great if there is an example about how to use columns with dashboard. Eg you have three columns, where you can drag&drop widgets between columns and inside current column.

**Comment by tarlepp** [comment]
Any update on this?

**Comment by andyperlitch** [comment]
Hey @tarlepp,

This is a significant departure from the current implementation, but that isn't to say that it is not possible. I would suggest trying to achieve this with a custom dashboard template. Otherwise you may need to fork and muck around with the core code.

Andy

**Comment by tarlepp** [comment]
In project where I'm trying to use this dashboard we will have about 30-40 different widgets and heights of those widgets are same about in 15 widgets. imho, this is one of the biggest problems with dashboard at current state. I cannot believe that everyone is using exactly same height on each widgets they are using. And yes, there is support for widget height change, but that will require huge amount of work to adjust each dashboard to look nice.

I already tried to use custom template (did about 2-3hours of experiment) to complish this but no luck. And that is why I opened this issue.

Basically it would be great (really awesome) if dashboard would support columns as in core feature, because I think it's the only way to make widgets more organized on dashboard and way that is easy to users to use really.

I can't imagine that I'm the only one who is/has facing this problem.

**Comment by andyperlitch** [comment]
So far you are the only one who has brought it up, and though I understand your use-case and sympathize with your problem, I don't have the time to explore this as a feature/change at the moment.

**Comment by tarlepp** [comment]
Good news!

Thanks to @sheppe fork of library. I just converted that to support newest version of original library and it works like a charm. (ruly there is some "small" issues though) But I'm going forward with this now.

Imho you should change ui-sortable to shapeshift because then this issue is resolved and basically original functionality won't change a bit.

Thanks for support.

**Comment by sheppe** [comment]
@tarlepp I'm glad you found my fork useful!

**Comment by tarlepp** [comment]
@sheppe Yeah, that was a really great help, although I did need to change some bits here and there, but concept works like a charm.

It would be nice if that library was default behaviour with dashboard. @andyperlitch What do you think about this?

**Comment by robertmazzo** [comment]
@tarlepp - Hi, is there a demo for the @sheppe fork of the dashboard widget framework ? I'm not sure I completely understand the column requirements you had back in March, but I'm interested in hearing more about it (as well as Shapeshift).
thanks,
Bob

**Comment by sheppe** [comment]
@robertmazzo I believe I updated /demo in the code to use shapeshift, so if you clone it you can check it out.

**Comment by tarlepp** [comment]
@robertmazzo I don't have a "live" example for you now :( basically I just fetched latest source from your repo and then make some "manual" adjustments according to @sheppe fork.
